presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java

```diff
@@ -312,6 +314,9 @@ private static void configureCompressionCodecs(JobConf jobConf)
             if (symlinkTarget && inputFormatClass == SymlinkTextInputFormat.class) {
                 // symlink targets are always TextInputFormat
                 inputFormatClass = TextInputFormat.class;
+                if (isDeserializerClass(schema, AvroSerDe.class)) {
```

Comment:
Would be good to update the comment above
```
// symlink targets are always TextInputFormat
```

to eg
```
// Symlink targets are assumed to be TEXTFILE unless serde indicates otherwise.
```